lib/expressions/Term.ts

```diff
@@ -98,7 +98,7 @@ export class Literal<T> extends Term {
   }
 
   public str(): string {
-    // eslint-disable-next-line @typescript-eslint/no-base-to-string
+    // @ts-expect-error TS2339
```

Comment:
Can we resolve this instead of ignoring it?

Reply:
I don't think so. There are I think 3 choices here
1. it just returns the basic string (as it is now)
2. We create our own basic string and check if the type has .toString or not
3. We just throw some error (I don't see any reason to pick this one)

:)

Reply:
Looks like an issue with the generic type `T` then. It should probably enforce the type to have a `toString()` method.

Reply:
Sure, I could solve that but that would make `NonLexicalLiteral` a little ugly.

Reply:
I'll change it in the next push and then you can see whether you like it.

Reply:
No changes?